contracts/modules/interfaces/IexecPoco2.v8.sol

```diff
@@ -0,0 +1,62 @@
+// SPDX-FileCopyrightText: 2020-2024 IEXEC BLOCKCHAIN TECH <contact@iex.ec>
+// SPDX-License-Identifier: Apache-2.0
+
+pragma solidity ^0.8.0;
+
+interface IexecPoco2 {
+    event AccurateContribution(address indexed worker, bytes32 indexed taskId);
+    event FaultyContribution(address indexed worker, bytes32 indexed taskId);
+
+    event TaskInitialize(bytes32 indexed taskId, address indexed workerpool);
+    event TaskContribute(bytes32 indexed taskId, address indexed worker, bytes32 hash);
+    event TaskConsensus(bytes32 indexed taskId, bytes32 consensus);
+    event TaskReveal(bytes32 indexed taskId, address indexed worker, bytes32 digest);
+    event TaskFinalize(bytes32 indexed taskId, bytes results);
+    event TaskClaimed(bytes32 indexed taskId);
+    event TaskReopen(bytes32 indexed taskId);
+
+    function initialize(bytes32 dealId, uint256 index) external returns (bytes32);
```

Comment:
How about more "natural" order?
- init
- initArray
- initClaimArray
- contrib
- contribFinalize
- reveal
- reopen
- finalize
- claim
- claimArray

(same for events)

Reply:
[Reorder interface functions](https://github.com/iExecBlockchainComputing/PoCo/pull/63/commits/ed7606ffe2ded05a91194382e6ba1b4d47eba318)